Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "concurrent-queue"
 # When publishing a new version:
 # - Update CHANGELOG.md
 # - Create "v1.x.y" git tag
-version = "1.2.4"
+version = "2.0.0"
 authors = ["Stjepan Glavina <stjepang@gmail.com>"]
 edition = "2018"
```

Comment:
Maybe update `authors`. We can't update edition due to the would-be-implied msrv bump

Reply:
I've added myself and @taiki-e to the `authors` list, since the GitHub contributors tab shows that we are the main contributors to this crate.